app/code/core/Mage/Sales/Block/Adminhtml/Billing/Agreement/View/Tab/Info.php

```diff
@@ -107,12 +109,10 @@ protected function _toHtml()
         );
         $this->setCustomerEmail($customer->getEmail());
         $this->setStatus($agreement->getStatusLabel());
-        $this->setCreatedAt(
-            $this->helper('core')->formatDate($agreement->getCreatedAt(), 'short', true)
-        );
+        $this->setCreatedAt($helper->formatDate($agreement->getCreatedAt(), 'short', true));
```

Comment:
Not sure if it's better:
```suggestion
        $this->setCreatedAt($this->formatDate($agreement->getCreatedAt(), 'short', true));
```